docs/FAQ/identity_management_FAQ.md

```diff
@@ -1,18 +1,18 @@
 ## Identity Management (Membership Service)
 &nbsp;
 ##### What is unique about the fabric's Membership Service module?
-The design and implementation of the Membership Service module encompass many of the latest advances in cryptography, which we believe make it stand apart from other alternatives.
+One of the things that makes the Membership Service module stand out from the pack is our implementation of the latest advances in cryptography.
 
-In addition to supporting generally-expected requirements such as preserving the privacy of transactions and making them auditable, the fabric's membership service module also introduces the concept of enrollment and transaction certificates. Infinite number of transaction certificates can be generated from their parent enrollment certificates (issued to validated users). This design ensures that asset tokens (which can be represented by transaction certificates) can only be created by verified owners, and private keys of asset tokens can be regenerated if lost.
+In addition to ensuring private, auditable transactions, our Membership Service module introduces the concept of enrollment and transaction certificates. This innovation ensures that only verified owners can create asset tokens, allowing an infinite number of transaction certificates to be issued through parent enrollment certificates while guaranteeing the private keys of asset tokens can be regenerated if lost. 
 
-Furthermore, the design of the system allows transaction certificates to both expire and be revoked, which allows issuers to have greater control over the asset tokens that they issued on a distributed chain.
+Issuers also have the ability revoke transaction certificates or designate them to expire within a certain timeframe, allowing greater control over the asset tokens they have issued. 
 
-Finally, like most other modules on the fabric, you can always replace the default membership service implementation with another option.
+Like most other modules on the fabric, you can always replace the default module with another membership service option should the need arise.
```

Comment:
I struggled with this line because it wasn't entirely clear to me whether it was trying to say that fabric users could replace the service module **entirely** or whether it was possible to change only certain functionalities. 

If it can be replaced entirely -- what other module frameworks are available? And if it's only possible to modify the framework, which capabilities are fungible and which are not? Also, who has the power to change the framework (whether entirely or in part) and how is that done?